### PR TITLE
7.0 add transifex config

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

**Comment by tafaRU** [comment]
@yvaucher the secret for the transifex password seems incomplete compared to what written in https://github.com/OCA/maintainer-quality-tools/issues/194

**Comment by yvaucher** [comment]
@tafaRU Indeed thanks

**Comment by yvaucher** [comment]
@tafaRU fixed

**Comment by tafaRU** [comment]
Now LGTM, thanks! :+1:

**Comment by nbessi** [comment]
@gurneyalex for version 7.0 we are missing https://github.com/OCA/bank-payment dependence to have a green runbot

**Comment by yvaucher** [comment]
@nbessi here we go for the missing dependency

**Comment by yvaucher** [comment]
Fixed a typo on use of --depth for git clone

**Comment by gurneyalex** [comment]
@nbessi fixed

File: .travis.yml [hunk 1]
```diff
@@ -6,35 +9,37 @@ addons:
...
     - sudo apt-get update -qq
     - sudo apt-get install -y python-xlrd
 
 install:
-  - git clone https://github.com/OCA/partner-contact-management ${HOME}/partner-contact-management -b ${VERSION}
-  - git clone https://github.com/OCA/bank-statement-reconcile ${HOME}/bank-statement-reconcile -b ${VERSION}
-  - git clone https://github.com/OCA/account-financial-tools ${HOME}/account-financial-tools -b ${VERSION}
-  - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
-  - git clone https://github.com/OCA/banking.git ${HOME}/banking -b ${VERSION}
+  - git clone --depth 1 https://github.com/OCA/partner-contact-management ${HOME}/partner-contact-management -b ${VERSION}
```
Comment (pedrobaeza) [inline .travis.yml]: Why not oca_dependencies.txt?

**Comment by nbessi** [comment]
@yvaucher I have rerun travis let see if it is green

**Comment by yvaucher** [comment]
Still red :disappointed:

**Comment by yvaucher** [comment]
rebased

**Comment by yvaucher** [comment]
```
2015-07-27 17:17:42,512 12869 INFO 3111686-144-a34148-all openerp.modules.loading: module l10n_ch: loading sterchi_chart/account.xml
2015-07-27 17:17:42,674 12869 WARNING 3111686-144-a34148-all openerp.osv.orm: No such field(s) in model account.account.type: partner_account.
2015-07-27 17:17:42,678 12869 WARNING 3111686-144-a34148-all openerp.osv.orm: No such field(s) in model account.account.type: partner_account.
2015-07-27 17:17:45,909 12869 WARNING 3111686-144-a34148-all openerp.osv.orm: No such field(s) in model account.chart.template: property_reserve_and_surplus_account.
```

**Comment by yvaucher** [comment]
Other WARNING are `Setting the`type`field is deprecated in the`ir.ui.view`model.` or come from other repositories.

**Comment by nbessi** [comment]
:+1:

**Pull request merged**